pkg/sloop/server/internal/config/config_test.go

```diff
@@ -0,0 +1,33 @@
+package config
+
+import (
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+	"encoding/json"
+	"github.com/ghodss/yaml"
+)
+
+func Test_loadFromJSONFile(t *testing.T) {
+	var expectedconfig SloopConfig
+	configfilename:="testconfig.json"
+	configFile, err := ioutil.ReadFile(configfilename)
+	err = json.Unmarshal(configFile, &expectedconfig)
+
```

Comment:
Add an assert here to make sure there is no error

Reply:
hi i agree we should thoroughly have checks to see if the file being passed is correct but i don't think they are part of this work item. i will create another work item for it though.

Reply:
The ask is to just add assert.Nil(t,err) at line 16. I don't think we should create WI for it.

Reply:
done